### DISCUSSION TITLE
Lifecycle Hooks to be run Outside the container?

### DISCUSSION THREAD

**Discussion by untamedspud** [containrrr/watchtower#1472] [Feature Requests] [answered]
I see I can do a pre- and post- script run **inside** the container before Watchtower performs updates.  
Is there any option to have a pre/post script run on the host **outside** rather than inside the container?

I'm running containers pulled from docker.io and yes, I guess I could copy or mount my scripts inside the container, which I'll do if you tell me that's the preferred and/or only method available.  Thanks in advance.

**Comment by simonaronsson** [comment, marked as answer]
Sorry, that's currently not possible.
